src/DistortableImageOverlay.js

```diff
@@ -211,16 +211,20 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
   },
 
   _cornerExceedsMapLats: function(zoom, corner, map) {
-    var exceedsTop;
-    var exceedsBottom;
-    if (zoom === 0) {
-      exceedsTop = map.project(corner).y < 2;
-      exceedsBottom = map.project(corner).y >= 255;
-    } else {
-      exceedsTop = map.project(corner).y / zoom < 2;
-      exceedsBottom = map.project(corner).y / Math.pow(2, zoom) >= 255;
+    if (map.options.crs == L.CRS.Simple) {
+      return false;
+    } else {  
```

Comment:
Aha, it's complaining in the automatic tests about trailing spaces. Try this quick fix!

```suggestion
    } else {
```